vit-servicing-station-lib/src/v0/endpoints/snapshot/routes.rs

```diff
@@ -9,17 +11,36 @@ pub fn filter(
 ) -> impl Filter<Extract = impl Reply, Error = Rejection> + Clone {
     let with_context = warp::any().map(move || context.clone());
 
-    let get_voting_power = warp::path!(String / String)
+    let get_voters_info = warp::path!(String / String)
         .and(warp::get())
         .and(with_context.clone())
-        .and_then(get_voting_power)
-        .boxed();
+        .and_then(get_voters_info);
 
     let get_tags = warp::path::end()
         .and(warp::get())
         .and(with_context)
-        .and_then(get_tags)
-        .boxed();
+        .and_then(get_tags);
 
-    root.and(get_voting_power.or(get_tags)).boxed()
+    root.and(get_voters_info.or(get_tags))
+}
+
+pub fn update_filter(
+    context: UpdateHandle,
```

Comment:
where is the `/api/v0/user/{id}` endpoint implemented ?

This seems to be snapshot related rather than user related - re `3367` ?

Reply:
This pr contains only refactor, I have not done any modification or update here. I have only moved code